service/storage_proxy.cc

```diff
@@ -123,6 +123,19 @@ seastar::metrics::label_instance current_scheduling_group_label() {
 
 }
 
+[[maybe_unused]] static locator::host_id convert_ep_to_hid_with_erm(locator::effective_replication_map_ptr ermptr,
```

Comment:
This name has the highest density of acronyms/contractions in the system. How is a newcomer supposed to understand it?